Sources/OHHTTPStubsSwift/OHHTTPStubsSwift.swift

```diff
@@ -73,6 +73,67 @@ import OHHTTPStubs
 
 // MARK: Syntaxic Sugar for OHHTTPStubs
 
+public extension HTTPStubsResponse {
+  /**
+   * Initializes an `HTTPStubsResponse` with a string and given arguments
+   *
+   * - Parameter string: string to be used as the response body
+   * - Parameter encoding: encoding used when creating data from a `string`
+   * - Parameter statusCode: HTTP status code for the response
+   * - Parameter headers: HTTP headers for the response
+   *
+   * - SeeAlso: `String.data(using:)`
+   */
+  convenience init(
+    string: String,
+    encoding: String.Encoding = .utf8,
+    statusCode: Int32 = 200,
+    headers: [AnyHashable: Any]? = nil
+  ) {
+    self.init(data: string.data(using: encoding)!, statusCode: statusCode, headers: headers)
+  }
+
+  /**
+   * Initializes an `HTTPStubsResponse` with a json object and given arguments
+   *
+   * - Parameter jsonObject: JSON object used as the response body, encoded with `JSONSerialization`
+   * - Parameter serializationOptions: encoding used when creating data from a `jsonObject`
+   * - Parameter statusCode: HTTP status code for the response
+   * - Parameter headers: HTTP headers for the response
+   *
+   * - SeeAlso: `JSONSerialization.data(withJSONObject:, options:)`
+   */
+  convenience init(
+    jsonObject: Any,
+    serializationOptions: JSONSerialization.WritingOptions = [],
+    statusCode: Int32 = 200,
+    headers: [AnyHashable: Any]? = nil
+  ) {
+    self.init(
+      data: try! JSONSerialization.data(withJSONObject: jsonObject, options: serializationOptions),
+      statusCode: statusCode,
+      headers: headers
+    )
+  }
+
+  /**
+   * Initializes an `HTTPStubsResponse` with an encodable object and given arguments
+   *
+   * - Parameter encodable: Encodable object used as the response body
+   * - Parameter encoder: `JSONEncoder` to encode the `encodable` object
+   * - Parameter statusCode: HTTP status code for the response
+   * - Parameter headers: HTTP headers for the response
+   */
+  convenience init<T>(
+    encodable: T,
+    encoder: JSONEncoder = .init(),
```

Comment:
consider removing this and documenting you're using a JSON encoder, or change the param name to `jsonEncoder`.